__tests__/index.test.ts

```diff
@@ -205,7 +205,28 @@ describe('Auth0', () => {
       expect(utils.runIframe).toHaveBeenCalled();
     });
 
-    it('should throw other errors that are not "login_required"', async () => {
+    it('should absorb "consent_required" errors', async () => {
```

Comment:
We should include tests here for the other errors that we want to absorb, not just `consent_required`.